src/app/shared/components/keyboard-input/keyboard-input.component.ts

```diff
@@ -33,6 +33,9 @@ export class KeyboardInputComponent implements ControlValueAccessor {
   @Input() public placeholder?: string
 
   @Input() public helpText: string
+  @Input() public errorMessage?: string
+  @Input() public isRequired?: boolean
+  @Input() public hasError?: boolean
```

Comment:
This attribute should be deleted. Refer to my comment on `TextInputComponent`.

Reply:
Fixed